### PR TITLE
Add Ombi Chart

### PR DESCRIPTION
**Pull request by onedr0p**
Please review and let me know if you need any changes.

Thanks!

Closes https://github.com/billimek/billimek-charts/issues/89

### CODE REVIEW

File: ombi/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# NZBGet Usenet client
```
Comment (billimek) [inline ombi/README.md]: This should read `Ombi` instead of `NZBGet`

File: ombi/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+|----------------------------|-------------------------------------|---------------------------------------------------------|
+| `image.repository`         | Image repository | `linuxserver/ombi` |
+| `image.tag`                | Image tag. Possible values listed [here](https://hub.docker.com/r/linuxserver/ombi/tags/).| `3.0.4914-ls72`|
+| `image.pullPolicy`         | Image pull policy | `IfNotPresent` |
+| `strategyType`             | Specifies the strategy used to replace old Pods by new ones | `Recreate` |
+| `timezone`                 | Timezone the Ombi instance should run as, e.g. 'America/New_York' | `UTC` |
+| `puid`                     | process userID the Ombi instance should run as | `1001` |
+| `pgid`                     | process groupID the Ombi instance should run as | `1001` |
+| `Service.type`          | Kubernetes service type for the Ombi GUI | `ClusterIP` |
+| `Service.port`          | Kubernetes port where the Ombi GUI is exposed| `6789` |
```
Comment (billimek) [inline ombi/README.md]: Should this be `3579`?

**Comment by onedr0p** [comment]
Updated

**Comment by billimek** [comment]
Thanks for this @onedr0p

**Pull request merged**